projects/quml-library/src/lib/section-player/section-player.component.html

```diff
@@ -30,13 +33,15 @@
               [contentName]="showRootInstruction ? parentConfig?.contentName : parentConfig?.isSectionsAvailable ? sectionConfig?.metadata?.name : parentConfig?.contentName">
             </quml-startpage>
           </slide>
+          <!--</ngx-carousel-item>-->
```

Comment:
remove the commented codes in all the places

Reply:
completed